schemas/common/all-types.json

```diff
@@ -4,5 +4,12 @@
 
   "title": "All",
   "description": "Any valid JSON data type.",
-  "type": ["array", "boolean", "null", "number", "object", "string"]
+
+  "oneOf": [
+    { "type": ["boolean", "null", "number", "string", "array"] },
+    {
+      "type": "object",
+      "minProperties": 2
```

Comment:
@mjenny 
@lina-salih 
We should also allow `object` with single keys if their name isn't an operator.

Reply:
done :)